app/src/main/java/com/example/drawingactivity/DialogUtils.kt

```diff
@@ -131,12 +163,18 @@ object DialogUtils {
                 }
                 DialogOperation.DELETE -> {
                     data.removeAt(index)
-                    recyclerView.adapter?.notifyItemChanged(index)
+                    recyclerView.adapter?.notifyItemRangeChanged(index, data.size)
```

Comment:
Why did this get changed to range? Delete should be one item at a time yes?

Reply:
Reverted to single item